src/modules/invite.ts

```diff
@@ -139,9 +139,12 @@ const inviteMany = async (
     const subject = 'Prytaneum Invite';
     // TODO Test with 1k invitees to handle Mailgun limit
     const subsetSize = 999;
-    for (let i = 0; i < inviteeList.length; i += subsetSize) {
+    for (let i = 0; i < inviteeList.length; i = subsetSize) {
```

Comment:
Is this a typo? `=` instead of `+=`

Reply:
Yes that's a typo, actually already fixed it but haven't pushed those changes yet. I'll push them all before tomorrow's meeting so you can go over them and we can discuss them at the meeting.